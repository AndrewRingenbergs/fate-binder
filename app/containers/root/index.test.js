import jest from 'jest';

import React from 'react';
import renderer from 'react-test-renderer';
import shallowRenderer from 'react-shallow-renderer';
import App from './index';

describe('<App />', ()=>{
  describe('for desktop mode', () => {
    global.matchMedia = () => {
      return {
        matches: false,
        addListener: () => {},
      }
    };

    test('should match snapshot', ()=>{
      const app = shallowRenderer(<App><p>Hello World</p></App>);
      expect(app).toMatchSnapshot();
    });
  });

  describe('for mobile mode', () => {
    global.matchMedia = () => {
      return {
        matches: true,
        addListener: () => {},
      }
    };

    test('should match snapshot', ()=>{
      const app = shallowRenderer(<App><p>Hello World</p></App>);
      expect(app).toMatchSnapshot();
    });
  });
});
