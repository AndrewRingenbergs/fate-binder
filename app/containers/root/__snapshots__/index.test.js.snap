exports[`<App /> should contain the name of the application 1`] = `
<div>
  <Connect(HeadingComponent) />
  <Connect(LoginComponent) />
</div>
`;
